library/src/main/java/com/medium/highlightjs/models/Language.kt

```diff
@@ -0,0 +1,125 @@
+package com.medium.highlightjs.models
+
+/**
+ * This Class was created by Patrick J
+ * on 09.06.16. For more Details and Licensing
+ * have a look at the README.md
+ */
+enum class Language(val languageName: String?) {
+    AUTO_DETECT(null),
+    DISABLE_HIGHLIGHT("nohighlight"),
+    _1C("1c"), ABNF("abnf"),
```

Comment:
Can the codestyle of this file be improved to have a language per line? It's hard to read it currently.